### DISCUSSION TITLE
To Committers and Contributors: 4.14 is pending.

### DISCUSSION THREAD

**Discussion by wimjongman** [eclipse-birt/birt#1254] [Announcements]
Hey All, I plan to release 4.14 as 4.13 does not install in the latest Eclipse. Let me know if there is something you are working on and want to include.

**Comment by speckyspooky** [comment]
Hi Wim, yes, the enhancement of #1224 would be great because there are some special changes also on rom.def side.

**Comment by hvbtup** [comment]
I'd appreciate if the RotatedTextItem sample plugin would be part of BIRT 4.14 build artifacts (e.g. directly usable in BIRT-All-In-One and at runtime).

**Comment by 858882847** [comment]
I also hope that Birt4.14 can include some plugins for text rotation that can be used directly at runtime without distinguishing them separately

**Comment by brgfsu** [comment]
Regarding the 4.14 release, has there been an update to the anticipated release date, tagging and packaging of 4.14?  The Eclipse project still states October 10th.

**Reply by wimjongman** [reply]
We have a blocker. https://github.com/eclipse-birt/birt/issues/1451

**Reply by brgfsu** [reply]
Thank you for the updated status.

**Comment by speckyspooky** [comment]
@wimjongman & @merks
Due to the usage of the new eclipse version 2023-12 which will be get the final release on December 6th we should directly afterwards close all changes and publish the BIRT 4.14.

Then we are 2 month later like the original plan of BIRT 4.14. I need really the new version because it is the base of my use platform.
But my supplier won't allow to use a BIRT-built without the "final status" / "release status".

@wimjongman What is your current plan?

**Reply by merks** [reply]
I think we are back on track. I am hopeful that WTP will get its house in order so that all jetty 10 dependencies are gone. I’ve focused my orbit work to ensure that WTP has updated dependencies available and that BIRT‘s dependencies are mostly updated and are functional.  It’s hard to update BIRT to some newer things when WTP drags in older things.  In any case, I am hopeful that we will only see further improvement. 

We should plan to have a release available on the same day as simrel. The rc phase later this month gives us time to finalize and to test.  I can help ensure that BIRT installs in 2023-12.  Together we can ensure that the runtimes work and are functionally complete.  The December 6th date is etched in stone and I’m confident we can deliver on that schedule.

**Reply by speckyspooky** [reply]
I would follow the suggestion of @merks.
Yes, it is hard for me to give one month again to the final BIRT 4.14 release but the argument of 2023-12 is a good argument.
So we would reduce the risk to have platform issues from there in our release.

**Reply by merks** [reply]
Note that I think there is a good chance we can have a release available/ready a week ahead of this schedule. That is what I’d like to aim for.  The dependencies are final for rc2 which is one week before the release.

**Reply by wimjongman** [reply]
All the blockers have been resolved, and AFAICT, we can release. I have done a smoke test on the latest nightlies, and all looks good.

**Reply by merks** [reply]
It's good we are in good shape, but to release with contents of the currently unreleased parts from 4.30/ 2023-12 and producing something that only properly installs in and works with 4.30 / 2023 seems like not a great plan to me.

**Comment by wimjongman** [comment]
I have set the release date to December 1 and I have scheduled a release review. which is tracked internally here:

 BIRT 4.14 Release Review #1478

**Reply by merks** [reply]
I think that is perfect!

**Reply by speckyspooky** [reply]
Sounds good for me, so I can plan the release on my side.

**Comment by wimjongman** [comment]
I have added milestone 4.15. Issues can be moved if we do not plan to fix them for 4.14.

**Comment by wimjongman** [comment]
We have a 4.14 milestone here:

https://download.eclipse.org/birt/updates/milestone/latest/index.html

Please download all components that you use and test them for your use case.

The targeted release date is December 6.
